lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -1815,6 +1845,48 @@ class SqlParser(private val ion: IonSystem) : Parser {
         }
     }
 
+    private fun List<Token>.parseLet(): ParseNode {
+        val letClauses = ArrayList<ParseNode>()
+        var rem = this.tail
+        var child = rem.parseExpression()
+        rem = child.remaining
+
+        if (rem.head?.type != AS) {
+            rem.head.err("Expected AS following LET expr", PARSE_EXPECTED_AS_FOR_LET)
+        }
+
+        rem = rem.tail
+
+        if (rem.head?.type?.isIdentifier() != true) {
+            rem.head.err("Expected identifier for $AS-alias", PARSE_EXPECTED_IDENT_FOR_ALIAS)
```

Comment:
nit: This error message uses `$AS` to interpolate, can we do the same for `LET` on line 1855? 
I do not mind either way as long as we are consistent

Reply:
Sure, also added to the other error message in the while loop.